tests/test_7_nsm_algae_calculations.py

```diff
@@ -0,0 +1,1909 @@
+    """
+    return GlobalParameters(
+        use_NH4= True,
+        use_NO3= True, 
+        use_OrgN= True,
+        use_OrgP = True,
+        use_TIP= True,  
+        use_SedFlux= False,
+        use_POC = True,
+        use_DOC = True,
+        use_DOX= True,
+        use_DIC= True,
+        use_Algae= True,
+        use_Balgae= True,
+        use_N2 = True,
+        use_Pathogen = True,
+        use_Alk = True,
+        use_POM = True 
+    )
+
+@pytest.fixture(scope='function')
+def default_gvars_params() -> GlobalVars:
+    """Returns default global variables static variable values for the model.
+
+    NOTE: As of now (3/18/2022) these match the built in defaults, but are 
+    copied here to allow for easy modification of the defaults in the future.
+
+    Returns a typed dictionary, with string keys and float values.
+    """
+    return GlobalVars(
+        vson = 0.01,
+        vsoc = 0.01,
+        vsop = 999,
+        vs = 999,
+        SOD_20 = 999,
+        SOD_theta = 999,
+        vb = 0.01,
+        fcom = 0.4,
+        kaw_20_user = 999,
+        kah_20_user = 999,
+        hydraulic_reaeration_option = 2,
+        wind_reaeration_option = 2,  
+        timestep = 1,    #TODO Dynamic or static?
+        depth = 1.5,         #TODO Dynamic or static?
+        TwaterC = 25,
+        theta = 1.047,
+        velocity = 1,
+        flow = 2,
+        topwidth = 1,
+        slope = 2,
+        shear_velocity = 4,
+        pressure_atm = 2,
+        wind_speed = 4,
+        q_solar = 500,
+        Solid = 1,
+        lambda0 = .02,
+        lambda1 = .0088,
+        lambda2 = .054,
+        lambdas = .052,
+        lambdam = .174, 
+        Fr_PAR = .47  
+    )
+
+def get_nutrient_budget_instance(
+    time_steps,
+    initial_nsm1_state,
+    default_algae_params,
+    default_alkalinity_params,
+    default_balgae_params,
+    default_nitrogen_params,
+    default_carbon_params,
+    default_CBOD_params,
+    default_DOX_params,
+    default_N2_params,
+    default_POM_params,
+    default_pathogen_params,
+    default_phosphorus_params,
+    default_gp_params,
+    default_gvars_params,
+
+) -> NutrientBudget:
+    """Return an instance of the NSM1 class."""
+    return NutrientBudget(
+        time_steps=time_steps,
+        initial_state_values=initial_nsm1_state,
+        algae_parameters=default_algae_params,
+        alkalinity_parameters=default_alkalinity_params,
+        balgae_parameters=default_balgae_params,
+        nitrogen_parameters=default_nitrogen_params,
+        carbon_parameters=default_carbon_params,
+        CBOD_parameters=default_CBOD_params,
+        DOX_parameters=default_DOX_params,
+        N2_parameters=default_N2_params,
+        POM_parameters=default_POM_params,
+        pathogen_parameters=default_pathogen_params,
+        phosphorus_parameters=default_phosphorus_params,
+        global_parameters=default_gp_params,
+        global_vars=default_gvars_params,
+        time_dim='nsm1_time_step',
```

Comment:
We get the comparison outputs from a model right -- what was the model timestep on that model? I think this may be important to consider with the time-dependent variables I flagged and tagged @aufdenkampe in.

Reply:
I set the timestep for the RAS model to 1 day and delta t to 86400 similar to the TSM test cases.